tests/misc_env/install_prereq.py

```diff
@@ -276,6 +276,7 @@ def setup_subscription_manager(ceph, server, timeout=300, interval=60):
                 serverurl=creds.get("serverurl"),
                 baseurl=creds.get("baseurl"),
                 force=True,
+                clean=True,
```

Comment:
Instead of this, please orchestrate the process of subscription manager registration and enabling it in a method. perform all required to enable manager via a method.